doc/source/getting_started/docker/index.rst

```diff
@@ -0,0 +1,61 @@
+.. _ref_docker:
+
+Geometry service using Docker
+=============================
+
+What is Docker?
+---------------
+
+Docker is an open platform for developing, shipping, and running apps in a
+containerized way.
+
+Containers are standard units of software that package up code and all its dependencies
+so the app runs quickly and reliably from one computing environment to another.
+
+Ensure that the machine where the Geometry service is to run has Docker installed. Otherwise,
+see `Install Docker Engine <https://docs.docker.com/engine/install/>`_ in the Docker documentation.
+
+Select your Docker container
+----------------------------
+
+Currently, the Geometry service backend is mainly delivered as a **Windows** Docker container.
+However, these containers require of a Windows machine to run them.
+
+A Linux version of the Geometry Service is also available but with limited capabilities,
+meaning that certain operations are not available or fail.
+
+Select the kind of Docker container you want to build:
+
+.. grid:: 1
+
+   .. grid-item-card:: Windows Docker Container
+            :link: windows_container
+            :link-type: doc
+            :margin: 2 2 0 0
+
+            Build a Windows Docker Container for the Geometry Service
+            and use it from PyAnsys Geometry. Exploit the full potential
+            of the Geometry Service.
```

Comment:
```suggestion
            Build a Windows Docker container for the Geometry service
            and use it from PyAnsys Geometry. Explore the full potential
            of the Geometry service.
```
Exploit has a rather negative connotation--hence the suggested change.

Reply:
Thanks for the suggestion! I thought it was mainly positive! 😄